modules/apps/object/object-test/src/testFunctional/macros/ObjectNotifications.macro

```diff
@@ -94,6 +94,18 @@ definition {
 			locator1 = "ObjectAdmin#TABLE_LIST_TITLE");
 	}
 
+	macro searchNotificationTemplate {
+		AssertElementPresent(locator1 = "ObjectPortlet#SEARCH_FIELD");
+
+		Type(
+			locator1 = "ObjectPortlet#SEARCH_FIELD",
+			value1 = "${entryName}");
+
+		KeyPress(
+			locator1 = "ObjectPortlet#SEARCH_FIELD",
+			value1 = "\RETURN");
+	}
+
```

Comment:
Since it's unlikely we'll re-use this macro, we can just add the functions to the test directly. We can always extract this in the future if needed.

Also, we don't need `AssertElementPresent` here since the test isn't checking for the search bar. We should avoid assertions that are not within the scope of the test.